### DISCUSSION TITLE
/posts/how-to-setup-wezterm-terminal

### DISCUSSION THREAD

**Discussion by giscus** [josean-dev/sveltekit-blog#16] [General]
# /posts/how-to-setup-wezterm-terminal

This is my step by step guide on how to setup Wezterm for an awesome, simple, feature-rich  & fast terminal setup for macOs

https://www.josean.com/posts/how-to-setup-wezterm-terminal

**Comment by curious22** [comment]
Thanks for useful article. Also, I like [fzf](https://github.com/junegunn/fzf) and fzf [plugin](https://github.com/ohmyzsh/ohmyzsh/tree/master/plugins/fzf) (oh-my-zsh) for tab completion and history search.

**Comment by snjadhav64** [comment]
I get a ghosting issue is I enable this option using all the settings in this post.

set -g default-terminal "tmux-256color"

**Comment by georgefromohio** [comment]
Would you be willing to do a video on [oh-my-posh](https://ohmyposh.dev/) since powerline might be eol in our life time? I've been trying to get it to work with `fzf` and `fzf-tab` along with `zsh-completions` but for some reason they don't all work together. I mean I can get them 90% there but I can't get certain completions like docker to work for some reason.. I'm not sure if it's because it's truly a `oh-my-zsh` plugin or what but any help would be appreciated! Thank you!

**Comment by prk-2023** [comment]
Thank you for a lovely article. 
Nicely done and well maintained flow.

I have setup wezterm  vertical/horizontal splits  +  re-size  splits with  customized key-binds to match Vim motion. 
This could be an alternative for folks that do not have tmux installed. or a way to generalize with same key-binds across  many applications.

**Comment by timmkent** [comment]
👏 AWESOME !!!!

**Comment by arya-vinayak** [comment]
How do I get the same to reflect on VS code

**Reply by mathmul** [reply]
Yes, icons aren't working in the code's integrated terminal, for example...

**Comment by hamburger-fries** [comment]
In the .zshrc file, there's an alias on line 101 that replaces the ls command with eza:

alias ls="eza --icons=always"

I see you did not include this in your guide.

brew install eza

**Comment by Hexacker** [comment]
I'm having a weird issue, I set up the font_size to be 24, when I save the config file, it works, when closing and opening Wezterm, it back to the default 16

**Comment by District089** [comment]
Can you please customize the tutorial for Raycast? thank you very much

**Comment by alecthegeek** [comment]
Awesome. I recently trashed my zsh config (even though I had it backed it up with Stow), so this was a great resource to get set up again.
